parcels/particleset.py

```diff
@@ -503,6 +513,58 @@ def lonlatdepth_dtype_from_field_interp_method(field):
                 return np.float64
         return np.float32
 
+    def convert_pset_to_dict(self, pfile, time, deleted_only=False):
```

Comment:
a simple rename to 'to_dict(self, ...)' I think would be nice.

Reply:
Ok! Changed in 0cfbf92.